hip-service/Discovery/Patient/Helpers/FuzzyNameMatcher.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+
+namespace hip_service.Discovery.Patient.Helpers
+{
+    public static class FuzzyNameMatcher
+    {
+        public static int LevenshteinDistance(string s1, string s2)
+        {
+            var string1 = s1 == null ? "" : s1.Trim().ToLower();
+            var string2 = s2 == null ? "" : s2.Trim().ToLower();
+            int l1 = string1.Length;
+            int l2 = string2.Length;
+            int[,] d = new int[l1 + 1, l2 + 1];
+
+            if (l1 == 0)
+            {
+                return l2;
+            }
+
+            if (l2 == 0)
+            {
+                return l1;
+            }
+
+            for (int i = 0; i <= l1; i++)
+                d[i, 0] = i;
+            for (int j = 0; j <= l2; j++)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4788558)